WcaOnRails/.env.development

```diff
@@ -5,6 +5,9 @@ GITHUB_CREATE_PR_ACCESS_TOKEN=
 STRIPE_API_KEY=sk_test_CY2eQJchZKUrPGQtJ3Z60ycA
 STRIPE_PUBLISHABLE_KEY=pk_test_N0KdZIOedIrP8C4bD5XLUxOY
 STRIPE_CLIENT_ID=ca_A2YDwmyOll0aORNYiOA41dzEWn4xIDS2
+PAYPAL_CLIENT_ID=AVOWeybtlJY6DJCDIu6LO64bVrzlsO9DiVFEFTnzraC3S5p4pJefSapwLUaRiroRinTEN8Wi9Phlg6On
+PAYPAL_CLIENT_SECRET=EIknLp919Gbuj2CYmEWECyKH5HwJTWQNuqFuCr1qFMrGNzwkF8dD0VkwzwI3P7LIjXeq_sGFElWaRxCj
```

Comment:
Assuming it's fine to just publish our sandbox credentials in a PR? Our stripe test creds are there?

Reply:
Yup, sandbox credentials are harmless by nature.